### DISCUSSION TITLE
[Example Request] Headless E-Commerce - Shopify Storefront API

### DISCUSSION THREAD

**Discussion by bitsmyth** [vercel/next.js#13807] [Ideas]
I dont know if you heared about [Shopify Storefront API](https://shopify.dev/docs/storefront-api) but as vercel released examples for most popular CMS´s, I can see headless e-commerce as a thing.

Shopify already got a [react-apollo-example](https://github.com/Shopify/storefront-api-examples/tree/master/react-apollo) so it only has to be implemented with SSR in mind.

What do you think about to generally roll out headless e-commerce examples?

**Comment by bitsmyth** [comment]
https://nextjs.org/commerce

THANKS VERCEL <3

**Reply by ajtran303** [reply]
This is clutch!
